config/jobs/kubernetes/sig-auth/signed-kubelet-server-certs.yaml

```
@@ -0,0 +1,46 @@
+periodics:
+- interval: 6h
+  name: ci-kubernetes-e2e-kind-signed-kubelet-certs
+  cluster: k8s-infra-prow-build
+  annotations:
+    testgrid-dashboards: sig-auth-kubelet-serving-certs
+    description: Runs conformance tests on a cluster with kubelet serving certificates signed by the CP against a latest kubernetes master cluster created with sigs.k8s.io/kind
+  labels:
+    preset-dind-enabled: "true"
+  decorate: true
+  decoration_config:
+    timeout: 60m
+  extra_refs:
+  - org: kubernetes
+    repo: kubernetes
+    base_ref: master
+    path_alias: k8s.io/kubernetes
```

this needs to be set as workdir?
```suggestion
  - org: kubernetes
    repo: kubernetes
    base_ref: master
    path_alias: k8s.io/kubernetes
    workdir: true
```